### ISSUE TITLE
Duplicating request params when using form data and array

### ISSUE THREAD

**Issue by PixeLInc** [spider-gazelle/action-controller#71]
When parsing form data that has an array of values for a field, every value gets duplicated N times due to a bug when parsing them from the body.

https://github.com/spider-gazelle/action-controller/blob/98798592c408c2e83746cbc17f2adf4e0c6a4c27/src/action-controller/body_parser.cr#L70-L79

test example showcasing the issue:
```cr
require "uri"

params = URI::Params.new
form_params = URI::Params.parse("field=1&field=2&field=3")
form_params.each do |key, _|
  values = params.fetch_all(key) || [] of String
  values.concat(form_params.fetch_all(key))
  params.set_all(key, values)
end

pp(params) #=> URI::Params{"field" => ["1", "2", "3", "1", "2", "3", "1", "2", "3"]}
```

**Comment by PixeLInc** [comment]
The example code is not mine, but exactly copied from the source of spider gazelle that I linked to above.

**Comment by stakach** [comment]
ahh, thanks for pointing that out, will fix 😅

**Comment by stakach** [comment]
fixed in version [v5.4.2](https://github.com/spider-gazelle/action-controller/releases/tag/v5.4.2)
